sass/_colors.scss

```diff
@@ -0,0 +1,53 @@
+$white: #fff;
+
+// US Web Design Standard colors
+$usa-gray-dark: #323a45;
+$usa-gray: #5b616b;
+
+$usa-gray-light: #aeb0b5;
+$usa-gray-lighter: #d6d7d9;
+$usa-gray-lightest: #f1f1f1;
+
+$usa-primary-darkest: #112e51;
+
+$usa-gold: #fdb81e;
+$usa-gold-light: #f9c642;
+$usa-gold-lighter: #fad980;
+
+$usa-green: #2e8540;
+$usa-green-light: #4aa564;
+$usa-green-lighter: #94bfa2;
+
+$usa-cool-blue: #205493;
+$usa-cool-blue-light: #4773aa;
+$usa-cool-blue-lighter: #8ba6ca;
+
+$usa-primary: #02bfe7;
```

Comment:
It looks like this is actually the `primary-alt` color, which I think is too bright. I would try `primary` which is #0071bc (if you're going to use this for the bars).